pmm-app/tests/verifyQAN_test.js

```diff
@@ -9,21 +10,21 @@ Before(async (I, qanPage, adminPage) => {
   await I.switchTo(qanPage.fields.iframe);
 });
 
-Scenario('Verify QAN Filter groups exist  @not-pr-pipeline', async (I, adminPage, qanPage) => {
+Scenario('Verify QAN Filter groups exist [critical] @not-pr-pipeline', async (I, adminPage, qanPage) => {
```

Comment:
I suggest to leave QAN tests for now, we already have a big mess in merging and it is taking me too much time to understand and merge and fix, could you revert this, we will add them when we enable all of them for 2.0

Reply:
yeah sure